hibernate-core/src/main/java/org/hibernate/loader/Loader.java

```diff
@@ -136,11 +135,22 @@ public Loader(SessionFactoryImplementor factory) {
 
 	/**
 	 * An array indicating whether the entities have eager property fetching
-	 * enabled.
+	 * enabled for all of their properties.
+	 * <p>
+	 * Supersedes {@link #getEntityPropertyEagerFetches()}.
 	 *
 	 * @return Eager property fetching indicators.
 	 */
-	protected boolean[] getEntityEagerPropertyFetches() {
```

Comment:
not sure if this renaming can cause any compatibility issue

Reply:
I agree.  As Loader is meant to be extended and this is protected it is at least possible that someone may override it